packages/vite/src/rsc/rscRequestHandler.ts

```diff
@@ -123,9 +126,44 @@ export function createRscRequestHandler() {
       }
 
       try {
+        // surround by performance, but also
+        // TODO (RSC): Add a way to turn this off in production
+        const startedAt = Date.now()
+        const start = performance.now()
         const pipeable = await renderRsc({ rscId, props, rsfId, args })
+        const endedAt = Date.now()
+        const end = performance.now()
+        const duration = end - start
```

Comment:
Just out of interest was it easier to use these different `.now` functions rather than having `startAt` come from the `performance.now`? Is `performance.now` difficult to turn back into a JS Date object? Just interested, don't mean to be nitpicker or anything

Reply:
I wanted to track start and end date times (proper timestamps) so startedAt, endedAt.

The performance start and end are some msec since Node starts something, so just used for the duration.

In the metadata I send to Studio, I send the startedAt, endedAt and duration:

```
      performance: {
        startedAt,
        endedAt,
        duration,
      },
```

but not those other time values -- because, yes, near impossible to transform back into date times.